# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libsigc++1/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN libsigc++-1.0.4-orig/tests/signals/convert_slot_test.cc libsigc++-1.0.4/tests/signals/convert_slot_test.cc
--- libsigc++-1.0.4-orig/tests/signals/convert_slot_test.cc	1999-07-04 01:00:40.000000000 +0200
+++ libsigc++-1.0.4/tests/signals/convert_slot_test.cc	2010-07-23 16:01:14.536518839 +0200
@@ -1,4 +1,5 @@
 #include <string>
+#include <cstring>
 #include <iostream>
 #include <sigc++/signal_system.h>
 #include <sigc++/convert.h>
